js/src/framework.ts

```diff
@@ -394,6 +400,33 @@ export async function runEvaluator(
   evaluator: EvaluatorDef<any, any, any, any>,
   progressReporter: ProgressReporter,
   filters: Filter[],
+): Promise<EvalResultWithSummary<any, any, any, any>> {
+  let result = runEvaluatorInternal(
+    experiment,
+    evaluator,
+    progressReporter,
+    filters,
+  );
+  let timer = async () => {
+    await new Promise((_, r) =>
+      setTimeout(() => {
```

Comment:
Would this just immediately run and throw an error if `evaluator.timeout` is undefined?

Reply:
Oops, fixed